demo/repl.html

```diff
@@ -46,6 +46,10 @@
 <script src="../third_party/CodeMirror-4.0.3/keymap/sublime.js"></script>
 
 <script src="../bin/traceur.js"></script>
+<script>
+window.System = new traceur.runtime.BrowserTraceurLoader();
+System.map = System.semverMap(System.version);
```

Comment:
What is the story about `System.map`? Is this the intended usage or are we just hacking this?